src/components/simulation-settings.scss

```diff
@@ -0,0 +1,173 @@
+@use "./vars.scss";
+
+.simulation-settings {
+  border: vars.$sim-border;
+  border-radius: vars.$sim-radius-settings-top vars.$sim-radius-settings-top vars.$sim-radius-settings-bottom vars.$sim-radius-settings-bottom;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  height: 306px;
+  position: relative;
+  width: 379px;
+
+  .settings-header {
+    align-items: center;
+    background-color: vars.$sim-dark-1;
+    border-top-left-radius: 3px;
+    border-top-right-radius: 3px;
+    box-sizing: border-box;
+    color: white;
+    display: flex;
+    height: 26px;
+    justify-content: center;
+    padding-bottom: 4px;
+    width: 100%;
+  }
+
+  .setting {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    margin: 13.5px 0 0 19px;
+    width: 208px;
+
+    .setting-title {
+      font-weight: bold;
+    }
+
+    .video-control-row {
+      display: flex;
+      margin: 8.5px 0 5px -10px;
+
+      .play-button {
+        background-image: url("../assets/new-sim/buttons/play-button-on.svg");
+        background-color: rgba(0, 0, 0, 0);
+        border: none;
+        height: 48px;
+        width: 48px;
+
+        &:hover {
+          background-image: url("../assets/new-sim/buttons/play-button-hover.svg");
+        }
+
+        &.playing {
+          background-image: url("../assets/new-sim/buttons/pause-button-on.svg");
+
+          &:hover {
+            background-image: url("../assets/new-sim/buttons/pause-button-hover.svg");
+          }
+        }
+      }
+
+      .slider-container {
+        align-items: center;
+        display: flex;
+        height: 48px;
+        margin-left: 37.5px;
+        width: 80px;
+
+        .rc-slider {
+          position: relative;
+          top: -7px;
+          width: 100%;
+          height: 14px;
+          padding: 12px 0;
+          border-radius: 2px;
+          touch-action: none;
+          box-sizing: border-box;
+          cursor: pointer;
+      
+          &-rail {
+            position: absolute;
+            width: 100%;
+            height: 3px;
+            background-color: vars.$sim-light-5;
+            border-radius: 2px;
+          }
+      
+          &-track {
+            position: relative;
+            width: 100%;
+            height: 3px;
+            background-color: vars.$sim-dark-1;
+            border-radius: 2px;
+          }
+      
+          &-handle {
+            background-color: vars.$sim-dark-1;
+            position: absolute;
+            top: -1px;
+            width: 26px;
+            height: 26px;
+            border-radius: 50%;
+            cursor: pointer;
+            cursor: grab;
+            touch-action: pan-x;
+            z-index: 2;
+      
+            &:focus {
+              outline: none;
+              box-shadow: none;
+            }
```

Comment:
This has the effect that I cannot tell when the slider is selected if I am tabbing through the controls. All the controls work nicely with the keyboard other than that; I would allow the slider to show its focus state.